stack/stack.go

```diff
@@ -1,39 +1,91 @@
 package main
 
+//Imported Libraries
 import (
-	"errors"
 	"fmt"
 )
 
-// Stack implements stack DS
-type Stack struct {
-	list []int
+//Constants
+const x  = 10
+
+//Implementation of the stack structure
+type stack struct {
+	items [x] string
+}
+//pop from the stack
+func (a *stack) push (value string){
+	var found bool = false
```

Comment:
should omit type bool from declaration of var found; it will be inferred from the right-hand side

*Origin: GoLintBear, Section: `golint`.*